openhcl/virt_mshv_vtl/src/processor/hardware_cvm/tlb_lock.rs

```diff
@@ -4,29 +4,34 @@
 //! TLB lock infrastructure support for hardware-isolated partitions.
 
 use crate::processor::HardwareIsolatedBacking;
+use crate::UhCvmPartitionState;
 use crate::UhProcessor;
 use hcl::GuestVtl;
 use hvdef::Vtl;
 use std::sync::atomic::Ordering;
 
-impl<B: HardwareIsolatedBacking> UhProcessor<'_, B> {
-    /// Causes the specified VTL on the current VP to wait on all TLB locks.
-    /// This is typically used to synchronize VTL permission changes with
-    /// concurrent instruction emulation.
+pub struct TlbLockAccess<'a> {
+    pub vp_index: usize,
```

Comment:
I'd really rather we use `VpIndex` consistently across virt code instead of a mix of `usize`, `u32`, and `VpIndex`.

Reply:
OK if you want to clean this up in a follow up.